.github/workflows/build.yml

```diff
@@ -105,6 +105,16 @@ jobs:
         run: |
           docker run -v $PWD:$PWD ${DOCKER_AWS_ENV} ${BASEMAPS_CONTAINER} import --config ${CONFIG_LOCATION} --commit
 
+      - name: (Prod - Config) Upload Latest Assets
+        if: github.ref == 'refs/heads/master' && github.event_name == 'push'
+        run: |
+          aws s3 cp ${ASSETS_HASH} s3://${BASEMAPS_BUCKET}/assets/assets-latest.tar.co
```

Comment:
the config should have a pointer to it's assets so I'm not sure we need this step?

Reply:
Yes, you are correct.  The config should already point to hashed assets. Just think to make this separated asset latest for the Default `AssetLocation` to use instead of static bucket?